src/Microsoft.AspNetCore.Blazor/Components/CascadingValue.cs

```diff
@@ -35,8 +36,18 @@ public class CascadingValue<T> : ICascadingValueComponent, IComponent
         /// </summary>
         [Parameter] private string Name { get; set; }
 
+        /// <summary>
+        /// If true, indicates that <see cref="Value"/> will not change. This is a
+        /// performance optimization that allows the framework to skip setting up
+        /// change notifications. Set this flag only if you will not change
+        /// <see cref="Value"/> during the component's lifetime.
+        /// </summary>
+        [Parameter] private bool Fixed { get; set; }
```

Comment:
My other idea for the name was `FixedValue` (or even `IsFixed`). It's a bit clearer, but there's a lot of repetition in the word `Value` if you have:

    <CascadingValue Value=something FixedValue=true>

@danroth27 @rynowak Got any preference? I'm happy either way.

Reply:
If you mind, let me drop my 2 cents here.
Value `IsValueFixed` seems to be best of words and very Microsofty in the name.
Another word which is used in UI word is `Frozen` so maybe `IsValueFrozen`. I rarely seen Fixed value in the context of UI frameworks.

Reply:
My vote is for `IsFixed`

Reply:
OK, it is fixed to `IsFixed`